ita-social-projects/GreenCityUser
# [Pickup city UBS.registered form] the title of the text and text message is not correct

## Summary Of Changes
Сhanged title and text message in email "confirmation email"


## Issue Link :clipboard:
[Link to the issue](https://github.com/ita-social-projects/GreenCity/issues/5211)

## Changed
Changed title and text message from "Green city UBS" to "Pick up city UBS"

## How to test :clipboard:
Make user registration
Go to mail
Open the letter and check if the title and text message have been changed to "Pick up city UBS"

# PR Checklist Forms

_(to be filled out by PR submitter)_
